spec/ama_layout/breadcrumb_builder_spec.rb

```diff
@@ -0,0 +1,47 @@
+describe AmaLayout::BreadcrumbBuilder do
+  let(:view_context) { ActionView::Base.new }
+
+  describe '#render' do
+    let(:builder) { AmaLayout::BreadcrumbBuilder.new(view_context, crumbs) }
+
+    context 'no separator specified' do
+      let(:crumbs) do
```

Comment:
What do you think about exposing the crumbs separately at the top, then you'd only have to override one of them in the disabled test?

```
  let(:foo_crumb) { BreadcrumbsOnRails::Breadcrumbs::Element.new('Foo', '/foo') }
  let(:bar_crumb) { BreadcrumbsOnRails::Breadcrumbs::Element.new('Bar', '/foo/bar') }
  let(:crumbs) { [foo_crumb, bar_crumb] }
```